zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/archiver/ElasticsearchArchiverRepository.java

```diff
@@ -238,24 +224,21 @@ private CompletableFuture<Void> setIndexLifeCycleToMatchingIndices(
     return client.indices().putSettings(settingsRequest).thenApplyAsync(ok -> null, executor);
   }
 
-  private ArchiveBatch createArchiveBatch(final SearchResponse<?> search) {
-    final var aggregate = search.aggregations().get(DATES_AGG);
-    if (aggregate == null) {
-      return null;
-    }
-
-    final List<DateHistogramBucket> buckets = aggregate.dateHistogram().buckets().array();
-    if (buckets.isEmpty()) {
-      return null;
+  private ArchiveBatch createArchiveBatch(final SearchResponse<?> response, final String field) {
```

Comment:
:wrench: parameter `field` should be `sortField` for clarity and to mirror the OS implementation